src/main/kotlin/racingcar/domain/Cars.kt

```diff
@@ -3,8 +3,15 @@ package racingcar.domain
 class Cars(carNames: List<String>) : List<Car> by (carNames.map { name -> Car(name) }) {
 
     fun winners(attemptCount: Int): List<Car> {
-        val maxPosition = maxByOrNull { car -> car.currentPosition(attemptCount) }
-            ?.currentPosition(attemptCount)
+        val maxPosition = map { car -> car.currentPosition(attemptCount) }.maxOrNull()
```

Comment:
아래의 코드로 개선해 보면 어떨까요?

```suggestion
        val maxPosition = map { it.currentPosition(attemptCount) }.maxOrNull()
```

Reply:
혹시 car라고 네이밍을 지정하는 것보다 it으로 사용하는게 더 추천되는 코드 스타일인가요?

Reply:
https://github.com/next-step/kotlin-racingcar/pull/238/commits/cb05ff6f2e154acabbad8ead96f3465b35cbe0ef 반영했습니다.